examples/apps/presence-tracker/src/MouseTracker.ts

```diff
@@ -18,98 +25,66 @@ export interface IMousePosition {
 	y: number;
 }
 
-export interface IMouseSignalPayload {
-	userId: string;
-	pos: IMousePosition;
-}
-
 export class MouseTracker extends TypedEventEmitter<IMouseTrackerEvents> {
-	private static readonly mouseSignalType = "positionChanged";
+	private readonly cursor: LatestValueManager<IMousePosition>;
 
 	/**
 	 * Local map of mouse position status for clients
 	 *
 	 * ```
-	 * Map<userId, Map<clientid, position>>
+	 * Map<ISessionClient, IMousePosition>
 	 * ```
 	 */
-	private readonly posMap = new Map<string, Map<string, IMousePosition>>();
-
-	private readonly onMouseSignalFn = (clientId: string, payload: IMouseSignalPayload) => {
-		const userId: string = payload.userId;
-		const position: IMousePosition = payload.pos;
-
-		let clientIdMap = this.posMap.get(userId);
-		if (clientIdMap === undefined) {
-			clientIdMap = new Map<string, IMousePosition>();
-			this.posMap.set(userId, clientIdMap);
-		}
-		clientIdMap.set(clientId, position);
-		this.emit("mousePositionChanged");
-	};
+	private readonly posMap = new Map<ISessionClient, IMousePosition>();
 
 	constructor(
+		public readonly presence: IPresence,
+		// eslint-disable-next-line @typescript-eslint/ban-types
+		statesWorkspace: PresenceStates<{}>,
 		public readonly audience: IAzureAudience,
-		private readonly signaler: ISignaler,
+		latencyInput: HTMLInputElement,
 	) {
 		super();
 
-		this.audience.on("memberRemoved", (clientId: string, member: IMember) => {
-			const clientIdMap = this.posMap.get(member.id);
-			if (clientIdMap !== undefined) {
-				clientIdMap.delete(clientId);
-				if (clientIdMap.size === 0) {
-					this.posMap.delete(member.id);
-				}
-			}
+		statesWorkspace.add("cursor", Latest({ x: 0, y: 0 }));
+		this.cursor = statesWorkspace.props.cursor;
+
+		latencyInput.addEventListener("input", (e) => {
+			const target = e.target as HTMLInputElement;
```

Comment:
I would move HTML things out of here. MouseTracker can have a method for setting.